aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,47 +1,26 @@
 ---
 remote_endpoint: ~
-name: "v1.11"
+name: "v1.12"
 proposals:
   - name: step_1_increase_max_txn_gas
     metadata:
       title: "Increase max txn gas temporarily for framework upgrade"
       description: "Increase max txn gas temporarily for framework upgrade"
     execution_mode: MultiStep
     update_sequence:
-      - DefaultGasWithOverrideOld:
+      - DefaultGasWithOverride:
+          feature_version: 15
```

Comment:
Should double check this. I think we're intending to bump to version 16 in release 1.11